backend/eventloop/shared_resource_loop/sharedresourceloop_managedenv_test.go

```diff
@@ -668,7 +666,69 @@ var _ = Describe("SharedResourceEventLoop Test", func() {
 
 		})
 
-		DescribeTable("Tests whether the tlsConfig value from managedEnv gets maped correctly into the database",
```

Comment:
The GitHub diff here is a bit weird, but the first part of the `DescribeTable` is unchanged in this PR.